src/providers/krb5/krb5_child_handler.c

```diff
@@ -556,6 +573,17 @@ static errno_t fork_child(struct tevent_context *ev,
         goto done;
     }
 
+    /* Setup child's keep alive timeout. This timeout is quite big to allow
+     * additional user interactions when the child is kept alive for further
+     * communication. */
```

Comment:
How does this keep alive timeout overlap with the krb5 auth timeout set in ``activate_child_timeout_handler()`` ?

Reply:
The timeout from activate_child_timeout_handler is active only for the single tevent request, it is disabled once the request is finished. The keep alive timeout is active as long as the file descriptors are opened. Does this answer your question?

Reply:
Yes, it makes sense.  Maybe it will be more clear by saying 'Setup child's keep alive timeout for open fds' in the comment

Reply:
Done.